CHANGELOG.md

```diff
@@ -1117,6 +1128,58 @@ Other minor changes
   _≐′_ : Pred A ℓ₁ → Pred A ℓ₂ → Set _
   ```
 
+* Added new proofs in `Relation.Unary.Properties`:
+  ```
+  ⊆-reflexive : Binary._⇒_ {A = Pred A ℓ₁} {B = Pred A ℓ₂} _≐_ _⊆_
```

Comment:
So to avoid overwhelming the reader with a gazillion quantifications and implicit arguments we usually just try and give the "essence of the type" in the CHANGELOG by removing implicits and simple universal quantifications. For example the first two entries would become:
```
⊆-reflexive : _≐_ ⇒ _⊆_
⊆-antisym : Antisymmetric _≐_ _⊆_
```